src/realm/db.hpp

```diff
@@ -634,6 +634,7 @@ class Transaction : public Group {
 
     // Live transactions state changes, often taking an observer functor:
     VersionID commit_and_continue_as_read(bool commit_to_disk = true) REQUIRES(!m_async_mutex);
+    void commit_and_continue_writing();
```

Comment:
There doesn't appear to be any problem with making this private and it's useful for keeping changesets under the server's size limit without semantic changes from releasing the lock.